openmdao/devtools/tests/test_viewconns.py

```diff
@@ -1,22 +1,22 @@
 import unittest
 
-from openmdao.api import Problem
-from openmdao.test_suite.components.sellar import SellarNoDerivatives
-from openmdao.api import view_connections
 
 class TestSellarFeature(unittest.TestCase):
 
-    def test_sellar(self):
-        # Just tests Newton on Sellar with FD derivs.
+    # no output checking, just make sure no exceptions raised
+    # Just tests Newton on Sellar with FD derivs.
+    def test_feature_sellar(self):
+        from openmdao.api import Problem
+        from openmdao.test_suite.components.sellar import SellarNoDerivatives
+        from openmdao.api import view_connections
```

Comment:
I think we want to use the new `import openmdao.api as om` pattern here since it is a feature test?